### DISCUSSION TITLE
빌드관련 질문입니다.

### DISCUSSION THREAD

**Discussion by Myeong-sup** [rinthel/opengl_course#5] [질문 & 답변]
![에러](https://user-images.githubusercontent.com/80265037/111899645-5829bd00-8a71-11eb-8e20-902ec7764634.PNG)

해결방법을 모르겠습니다.

**Comment by rinthel** [comment]
@Myeong-sup  로그 전체를 긁어서 여기다 붙여주세요

**Reply by Myeong-sup** [reply]
[build] 빌드를 시작하는 중
[proc] 명령 실행 중: "C:\Program Files\CMake\bin\cmake.EXE" --build c:/Cmake/work/cg/CMAKE_PROJECT_EXAMPLE/build --config Debug --target ALL_BUILD -- /maxcpucount:18
[build] .NET Framework용 Microsoft (R) Build Engine 버전 16.9.0+57a23d249
[build] Copyright (C) Microsoft Corporation. All rights reserved.
[build] 
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018: "GetOutOfDateItems" 작업에서 예기치 않은 오류가 발생했습니다. [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018: System.TypeInitializationException: 'Microsoft.Build.Utilities.FileTracker'의 형식 이니셜라이저에서 예외를 Throw했습니다. ---> System.IO.FileNotFoundException: 지정된 파일을 찾을 수 없습니다. (예외가 발생한 HRESULT: 0x80070002) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018:    위치: System.Runtime.InteropServices.Marshal.ThrowExceptionForHRInternal(Int32 errorCode, IntPtr errorInfo) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018:    위치: Microsoft.Build.Shared.NativeMethodsShared.ThrowExceptionForErrorCode(Int32 errorCode) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018:    위치: Microsoft.Build.Shared.NativeMethodsShared.GetShortFilePath(String path) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018:    위치: Microsoft.Build.Utilities.FileTracker..cctor() [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018:    --- 내부 예외 스택 추적의 끝 --- [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018:    위치: Microsoft.Build.Utilities.DependencyTableCache.FormatNormalizedTlogRootingMarker(ITaskItem[] tlogFiles) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018:    위치: Microsoft.Build.Utilities.CanonicalTrackedOutputFiles.ConstructOutputTable() [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018:    위치: Microsoft.Build.Utilities.CanonicalTrackedOutputFiles.InternalConstruct(ITask ownerTask, ITaskItem[] tlogFiles, Boolean constructOutputsFromTLogs) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018:    위치: Microsoft.Build.CPPTasks.GetOutOfDateItems.Execute() [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018:    위치: Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(202,5): error MSB4018:    위치: Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext() [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\ZERO_CHECK.vcxproj]
[build] 빌드가 완료됨(종료 코드: 1)

**Comment by rinthel** [comment]
@Myeong-sup builid 디렉토리를 제거하고 cmake: configure 부터 다시 진행해보세요

**Reply by Myeong-sup** [reply]
[main] 폴더를 구성하는 중: CMAKE_PROJECT_EXAMPLE 
[proc] 명령 실행 중: "C:\Program Files\CMake\bin\cmake.EXE" --no-warn-unused-cli -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=TRUE -Hc:/Cmake/work/cg/CMAKE_PROJECT_EXAMPLE -Bc:/Cmake/work/cg/CMAKE_PROJECT_EXAMPLE/build -G "Visual Studio 16 2019" -T host=x64 -A x64
[cmake] Not searching for unused variables given on the command line.
[cmake] -- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19042.
[cmake] CMake Error at CMakeLists.txt:6 (project):
[cmake]   Failed to run MSBuild command:
[cmake] 
[cmake]     C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/MSBuild/Current/Bin/MSBuild.exe
[cmake] 
[cmake]   to get the value of VCTargetsPath:
[cmake] 
[cmake]     .NET Framework용 Microsoft (R) Build Engine 버전 16.9.0+57a23d249
[cmake]     Copyright (C) Microsoft Corporation. All rights reserved.
[cmake]     
[cmake]     빌드 시작: 2021-03-22 오후 5:26:00
[cmake]     1 노드의 "C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj" 프로젝트(기본 대상)입니다.
[cmake]     PrepareForBuild:
[cmake]       "x64\Debug\" 디렉터리를 만들고 있습니다.
[cmake]       "x64\Debug\VCTargetsPath.tlog\" 디렉터리를 만들고 있습니다.
[cmake]     InitializeBuildStatus:
[cmake]       "AlwaysCreate"이(가) 지정되었기 때문에 "x64\Debug\VCTargetsPath.tlog\unsuccessfulbuild"을(를) 만들고 있습니다.
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003: 지정한 작업 실행 파일 "cmd.exe"을(를) 실행할 수 없습니다. System.IO.IOException: 임시 파일을 만들지 못했습니다. 임시 파일 폴더가 꽉 찼거나 경로가 올바르지 않습니다. 'C:\Users\?????\AppData\Local\Temp\' 경로에 대한 액세스가 거부되었습니다. ---> System.UnauthorizedAccessException: 'C:\Users\?????\AppData\Local\Temp\' 경로에 대한 액세스가 거부되었습니다. [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake] -- Configuring incomplete, errors occurred!
[cmake] See also "C:/Cmake/work/cg/CMAKE_PROJECT_EXAMPLE/build/CMakeFiles/CMakeOutput.log".
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: System.IO.Directory.InternalCreateDirectory(String fullPath, String path, Object dirSecurityObj, Boolean checkHost) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: System.IO.Directory.InternalCreateDirectoryHelper(String path, Boolean checkHost) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: System.IO.Directory.CreateDirectory(String path) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: Microsoft.Build.Shared.FileUtilities.GetTemporaryFile(String directory, String extension, Boolean createFile) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    --- 내부 예외 스택 추적의 끝 --- [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: Microsoft.Build.Shared.FileUtilities.GetTemporaryFile(String directory, String extension, Boolean createFile) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: Microsoft.Build.Tasks.Exec.CreateTemporaryBatchFile() [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: Microsoft.Build.Tasks.Exec.AddCommandLineCommands(CommandLineBuilderExtension commandLine) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: Microsoft.Build.Tasks.ToolTaskExtension.GenerateCommandLineCommands() [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: Microsoft.Build.Utilities.ToolTask.Execute() [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     "C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj" 프로젝트를 빌드했습니다(기본 대상). - 실패
[cmake]     
[cmake]     빌드하지 못했습니다.
[cmake]     
[cmake]     "C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj"(기본 대상)(1)->
[cmake]     (PostBuildEvent 대상) -> 
[cmake]       C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003: 지정한 작업 실행 파일 "cmd.exe"을(를) 실행할 수 없습니다. System.IO.IOException: 임시 파일을 만들지 못했습니다. 임시 파일 폴더가 꽉 찼거나 경로가 올바르지 않습니다. 'C:\Users\?????\AppData\Local\Temp\' 경로에 대한 액세스가 거부되었습니다. ---> System.UnauthorizedAccessException: 'C:\Users\?????\AppData\Local\Temp\' 경로에 대한 액세스가 거부되었습니다. [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: System.IO.Directory.InternalCreateDirectory(String fullPath, String path, Object dirSecurityObj, Boolean checkHost) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: System.IO.Directory.InternalCreateDirectoryHelper(String path, Boolean checkHost) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: System.IO.Directory.CreateDirectory(String path) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: Microsoft.Build.Shared.FileUtilities.GetTemporaryFile(String directory, String extension, Boolean createFile) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    --- 내부 예외 스택 추적의 끝 --- [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: Microsoft.Build.Shared.FileUtilities.GetTemporaryFile(String directory, String extension, Boolean createFile) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: Microsoft.Build.Tasks.Exec.CreateTemporaryBatchFile() [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: Microsoft.Build.Tasks.Exec.AddCommandLineCommands(CommandLineBuilderExtension commandLine) [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: Microsoft.Build.Tasks.ToolTaskExtension.GenerateCommandLineCommands() [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB6003:    위치: Microsoft.Build.Utilities.ToolTask.Execute() [C:\Cmake\work\cg\CMAKE_PROJECT_EXAMPLE\build\CMakeFiles\3.20.0-rc3\VCTargetsPath.vcxproj]
[cmake]     
[cmake]         경고 0개
[cmake]         오류 1개
[cmake]     
[cmake]     경과 시간: 00:00:00.20
[cmake]     
[cmake] 
[cmake]   Exit code: 1
[cmake] 
[cmake] 
[cmake] 

빌드를 제거하고 다시해봤는데 이렇게 됬습니다.

**Comment by rinthel** [comment]
@Myeong-sup 지금 붙여넣은 로그중에 경로에 ????가 찍혀있는게 보이네요. 아마도 윈도우 계정 이름이 한글로 되어있어서 생기는 문제인거 같습니다.
c:\dev\cmake_project_example 처럼 경로명 중간에 한글이 끼지 않도록 프로젝트를 이동한다음에 다시 시도해보세요

**Reply by Myeong-sup** [reply]
감사합니다 해결이 되었습니다.

**Reply by Myeong-sup** [reply]
[main] 폴더를 빌드하는 중: first_opengl_example 
[build] 빌드를 시작하는 중
[proc] 명령 실행 중: "C:\Program Files\CMake\bin\cmake.EXE" --build c:/dev/work/cg/first_opengl_example/build --config Debug --target ALL_BUILD -- /maxcpucount:18
[build] .NET Framework용 Microsoft (R) Build Engine 버전 16.9.0+57a23d249
[build] Copyright (C) Microsoft Corporation. All rights reserved.
[build] 
[build]   Performing build step for 'dep_glad'
[build]   .NET Framework용 Microsoft (R) Build Engine 버전 16.9.0+57a23d249
[build]   Copyright (C) Microsoft Corporation. All rights reserved.
[build]   
[build]     Generating GLAD
[build]     '_Python_EXECUTABLE-NOTFOUND'은(는) 내부 또는 외부 명령, 실행할 수 있는 프로그램, 또는
[build]     배치 파일이 아닙니다.
[build] C:\VSCODE\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(240,5): error MSB8066: 'C:\dev\work\cg\first_opengl_example\build\dep_glad-prefix\src\dep_glad-build\CMakeFiles\862c10906c535b9c9689241c16cc3589\glad.c.rule;C:\dev\work\cg\first_opengl_example\build\dep_glad-prefix\src\dep_glad-build\CMakeFiles\2c9526cbd6c64a6a82de855409a5a705\glad-generate-files.rule'에 대한 사용자 지정 빌드가 종료되었습니다(코드 9009). [C:\dev\work\cg\first_opengl_example\build\dep_glad-prefix\src\dep_glad-build\glad-generate-files.vcxproj] [C:\dev\work\cg\first_opengl_example\build\dep_glad.vcxproj]
[build] C:\VSCODE\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(240,5): error MSB8066: 'C:\dev\work\cg\first_opengl_example\build\CMakeFiles\369a873695faf121422b65dee6f0a113\dep_glad-build.rule;C:\dev\work\cg\first_opengl_example\build\CMakeFiles\369a873695faf121422b65dee6f0a113\dep_glad-install.rule;C:\dev\work\cg\first_opengl_example\build\CMakeFiles\369a873695faf121422b65dee6f0a113\dep_glad-test.rule;C:\dev\work\cg\first_opengl_example\build\CMakeFiles\d6c74ac5b73e4ddca5c339eff794fb84\dep_glad-complete.rule;C:\dev\work\cg\first_opengl_example\build\CMakeFiles\7074e463fde94aaf5e8dce405db80fe5\dep_glad.rule'에 대한 사용자 지정 빌드가 종료되었습니다(코드 1). [C:\dev\work\cg\first_opengl_example\build\dep_glad.vcxproj]
[build] 빌드가 완료됨(종료 코드: 1)

교수님 이건 무슨 오류인지 잘모르겠습니다.

**Reply by Myeong-sup** [reply]
build - install에 glad가 생성되지가 않습니다.

**Reply by rinthel** [reply]
[build] '_Python_EXECUTABLE-NOTFOUND'은(는) 내부 또는 외부 명령, 실행할 수 있는 프로그램, 또는
[build] 배치 파일이 아닙니다

이 에러는 파이썬이 설치되어있지 않아서 발생하는 문제입니다. 파이썬을 설치하고 다시 해보세요. 이루리의 강의자료에 해결책이 적혀있습니다
